packages/@lwc/engine-dom/src/initializeRenderer.ts

```diff
@@ -5,126 +5,8 @@
-    setGetFirstElementChild,
-    setGetLastChild,
-    setGetLastElementChild,
-    setGetProperty,
-    setHTMLElement,
-    setInsert,
-    setIsConnected,
-    setIsHydrating,
-    setIsNativeShadowDefined,
-    setIsSyntheticShadowDefined,
-    setNextSibling,
-    setQuerySelector,
-    setQuerySelectorAll,
-    setRemove,
-    setRemoveAttribute,
-    setRemoveEventListener,
-    setSetAttribute,
-    setSetCSSStyleProperty,
-    setSetProperty,
-    setSetText,
-    setSsr,
-    setAddEventListener,
-    setInsertStylesheet,
-} from '@lwc/engine-core';
+import { setDefaultRenderer } from '@lwc/engine-core';
 
-import {
-    assertInstanceOfHTMLElement,
-    attachShadow,
-    createComment,
-    createElement,
-    createText,
-    defineCustomElement,
-    dispatchEvent,
-    getAttribute,
-    getBoundingClientRect,
-    getChildNodes,
-    getChildren,
-    getClassList,
-    getCustomElement,
-    getElementsByClassName,
-    getElementsByTagName,
-    getFirstChild,
-    getFirstElementChild,
-    getLastChild,
-    getLastElementChild,
-    getProperty,
-    HTMLElement,
-    insert,
-    isConnected,
-    isHydrating,
-    isNativeShadowDefined,
-    isSyntheticShadowDefined,
-    nextSibling,
-    querySelector,
-    querySelectorAll,
-    remove,
-    removeAttribute,
-    removeEventListener,
-    setAttribute,
-    setCSSStyleProperty,
-    setProperty,
-    setText,
-    ssr,
-    addEventListener,
-    insertStylesheet,
-} from './renderer';
+import * as renderer from './renderer';
 
-setAssertInstanceOfHTMLElement(assertInstanceOfHTMLElement);
-setAttachShadow(attachShadow);
-setCreateComment(createComment);
-setCreateElement(createElement);
-setCreateText(createText);
-setDefineCustomElement(defineCustomElement);
-setDispatchEvent(dispatchEvent);
-setGetAttribute(getAttribute);
-setGetBoundingClientRect(getBoundingClientRect);
-setGetChildNodes(getChildNodes);
-setGetChildren(getChildren);
-setGetClassList(getClassList);
-setGetCustomElement(getCustomElement);
-setGetElementsByClassName(getElementsByClassName);
-setGetElementsByTagName(getElementsByTagName);
-setGetFirstChild(getFirstChild);
-setGetFirstElementChild(getFirstElementChild);
-setGetLastChild(getLastChild);
-setGetLastElementChild(getLastElementChild);
-setGetProperty(getProperty);
-setHTMLElement(HTMLElement);
-setInsert(insert);
-setIsConnected(isConnected);
-setIsHydrating(isHydrating);
-setIsNativeShadowDefined(isNativeShadowDefined);
-setIsSyntheticShadowDefined(isSyntheticShadowDefined);
-setNextSibling(nextSibling);
-setQuerySelector(querySelector);
-setQuerySelectorAll(querySelectorAll);
-setRemove(remove);
-setRemoveAttribute(removeAttribute);
-setRemoveEventListener(removeEventListener);
-setSetAttribute(setAttribute);
-setSetCSSStyleProperty(setCSSStyleProperty);
-setSetProperty(setProperty);
-setSetText(setText);
-setSsr(ssr);
-setAddEventListener(addEventListener);
-setInsertStylesheet(insertStylesheet);
+setDefaultRenderer(renderer);
```

Comment:
this is great because it means that the `renderer` object is really never leaked out. This might poses a problem for LWS, but we will see @ravijayaramappa

Reply:
not longer the case...